pkg/pxc/users/users.go

```diff
@@ -0,0 +1,76 @@
+package users
+
+import (
+	"database/sql"
+
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/pkg/errors"
+)
+
+type Manager struct {
+	db *sql.DB
+}
+
+type SysUser struct {
+	Name  string   `yaml:"username"`
+	Pass  string   `yaml:"password"`
+	Hosts []string `yaml:"hosts"`
+}
+
+func NewManager(host string, rootPass string) (Manager, error) {
+	var um Manager
+
+	mysqlDB, err := sql.Open("mysql", "root:"+rootPass+"@tcp("+host+")/?interpolateParams=true")
+	if err != nil {
+		return um, errors.Wrap(err, "cannot connect to any host")
+	}
+	um.db = mysqlDB
+
+	return um, nil
+}
+
+func (u *Manager) UpdateUsersPass(users []SysUser) error {
+	defer u.db.Close()
+	tx, err := u.db.Begin()
+	if err != nil {
+		return errors.Wrap(err, "begin transaction")
+	}
+
+	_, err = tx.Exec("FLUSH PRIVILEGES")
+	if err != nil {
+		errT := tx.Rollback()
+		if errT != nil {
+			return errors.Errorf("flush privileges: %v, tx rollback: %v", err, errT)
+		}
+		return errors.Wrap(err, "flush privileges")
+	}
```

Comment:
why do we need to flush privileges two times in one transaction? I think only the second run in the transaction has a real effect.

Reply:
removed https://github.com/percona/percona-xtradb-cluster-operator/pull/418/commits/3c3d08534e84f25c5701e9a930f3dc3783fcac9e